Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.9.7
```

Comment:
Thanks for separating out changes into little pull requests! Now that the version change has been merged, you can `git rebase upstream/master` and then `git push --force` and this commit should no longer be visible here.